src/audio.c

```diff
@@ -195,29 +195,29 @@ void func_80000968(s32 arg0) {
     switch (arg0) {
         case 1:
             func_80004A60(0, 0);
-            func_80004A60(1, 0x7FFF);
+            func_80004A60(1, 32767);
             func_80004A60(2, 0);
             func_80004A60(4, 0);
             alCSPSetVol(gMusicPlayer, (s16)(musicRelativeVolume * musicVolumeSliderPercentage >> 2));
             alCSPSetVol(gSndFxPlayer, 0);
             break;
         case 2:
             func_80004A60(0, 0);
-            func_80004A60(1, 0x7FFF);
-            func_80004A60(2, 0x7FFF);
-            func_80004A60(4, 0x7FFF);
+            func_80004A60(1, 32767);
```

Comment:
Food for thought. Do we need a constant for something like `MAX_S16_VAL`? These values seem too specific. Don't need to change the code, just something to think about.